src/classes/RLLP_OppContactRollup_BATCH.cls

```diff
@@ -48,7 +51,7 @@ public class RLLP_OppContactRollup_BATCH implements Database.Batchable<SObject>,
     }
     
     /*******************************************************************************************************
-    * @description Batch start method, instantiates RLLP_OppRollup_UTIL and builds query for contact rollups.
+    * @description Batch start method and builds query for contact rollups.
     ********************************************************************************************************/
     public database.Querylocator start(Database.BatchableContext bc) {
         RLLP_OppRollup_UTIL oppRollupUtil = new RLLP_OppRollup_UTIL();
```

Comment:
since you now instantiate oppRollupUtil as a member variable on line 40 above, do you still need to do it here?

Reply:
Nope, let me change that as well. I had some versioning issues with this class for some reason. Thanks!